wunderio/client-UK-FSA-beta
# FSA-1022: fix misaligned delivery options tab

To test:
- From the navbar, go to `News and alerts > Subscribe to news and alerts`
- The 'Delivery options' tab should be inline with the other two tabs, but not a link (black text; no hover effect)
- After picking some subscription options and moving to the next stage, it should now be a link and match the others
